src/mito2/src/request.rs

```diff
@@ -693,7 +693,7 @@ pub(crate) struct CompactionFinished {
     /// Region id.
     pub(crate) region_id: RegionId,
     /// Compaction result senders.
-    pub(crate) senders: Vec<OutputTx>,
+    pub(crate) senders: Option<Vec<OutputTx>>,
```

Comment:
Why do we need to wrap the `Vec` by `Option`? We can use an empty vec if there is no sender.

Reply:
oops, you're right. Let me change that.